### DISCUSSION TITLE
Log: time format of the log file

### DISCUSSION THREAD

**Discussion by jungyeon0904** [tendermint/tendermint#7140] [Q&A]
### My Question:
How do I change the time format of the log?
I want to see it by millisecond time, but I can't.
Please tell me how to change the time format of the log file.

### My Node Version:
gaia: v5.0.5
tendermint: v0.34.9

### My Node Log(level: Debug | type: json):
{"level":"debug","module":"consensus","chId":32,"msg":{"Height":8009665,"Round":0,"Type":2,"Index":63},"src":{"Logger":{},"Data":{}},**"time":"2021-10-15T11:35:39+09:00"**,"message":"Receive"}
{"level":"debug","module":"consensus","chId":34,"msg":{"Vote":{"type":2,"height":8009665,"round":0,"block_id":{"hash":"5D5D2D29B42AD6F1553A02D79500D4540EC93D83D7D412FDA0F4325C81B0B4C3","parts":{"total":1,"hash":"1E46278FD42AE48C2512E6182D0557F9FC54C3C016DD6A886E0FB193B5559939"}},"timestamp":"2021-10-15T02:35:36.893262445Z","validator_address":"46A3F8B8393BAA153C40E5722EAE82EA0D48B32D","validator_index":63,"signature":"CzpoZvDMdU5hl3V04f7Gn9eHspzkQCs6z3orMHyttw7gl7ykcPNr+c4DaUJK3nojSmvp/JYBt0MBRiiqeKGEAw=="}},"src":{"Logger":{},"Data":{}},**"time":"2021-10-15T11:35:39+09:00"**,"message":"Receive"}
{"level":"debug","module":"consensus","H/R":"8009665/0","peerH/R":"8009666/0","index":63,"type":2,**"time":"2021-10-15T11:35:39+09:00"**,"message":"setHasVote"}
{"level":"debug","module":"consensus","cs_height":8009666,"val_index":63,"vote_height":8009665,"vote_type":2,**"time":"2021-10-15T11:35:39+09:00"**,"message":"adding vote"}
{"level":"debug","module":"consensus","last_commit":"VoteSet{H:8009665 R:0 T:SIGNED_MSG_TYPE_PRECOMMIT +2/3:5D5D2D29B42AD6F1553A02D79500D4540EC93D83D7D412FDA0F4325C81B0B4C3:1:1E46278FD42A(1) BA{150:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx} map[]}",**"time":"2021-10-15T11:35:39+09:00"**,"message":"added vote to last precommits"}

### Homepage Log:
**I[10-04|13:54:30.393]** enterNewRound(91/0). Current: 91/0/RoundStepNewHeight module=consensus
**I[10-04|13:54:30.393]** enterPropose(91/0). Current: 91/0/RoundStepNewRound module=consensus
**I[10-04|13:54:30.393]** enterPropose: Our turn to propose            module=consensus proposer=125B0E3C5512F5C2B0E1109E31885C4511570C42 privValidator="PrivValidator{125B0E3C5512F5C2B0E1109E31885C4511570C42 LH:90, LR:0, LS:3}"

**Comment by creachadair** [comment]
There isn't currently a configuration setting for the time format. You can specify the log level and whether the output is plain text or JSON, but other aspects of the log format are not currently exposed.
